grunttasks/l10n-htmllint.js

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  grunt.registerTask('l10n-htmllint', 'Lint l10n files', function () {
+    // grunt addition for htmllint
+  });
+
+  grunt.registerMultiTask('l10n-json-to-html', 'Convert l10n JSON files to HTML', function () {
+    var done = this.async();
+    this.files.forEach(function (file) {
+      var content = '';
+      var src = file.src[0];
+      var path = src.split('/');
+      var locale = grunt.file.readJSON(src);
+      for (var val in locale) {
+        var value = locale[val];
+        if (typeof value !== 'object' && value !== '' && isNaN(value)) {
+          console.log(value);
+          content += value.toString() + '\n\n';
+        }
+        // on second thought, doesnt seem to be required, objects dont contain translations anyway
+        // if (typeof locale[val] === 'object') {
+        //  content += JSON.stringify(locale[val]);
+        // }
+        grunt.file.write('.tmp/' + path[1] + '/' + path[2] + '/' + path[3].replace('json', 'html'), content);
```

Comment:
We may want to do a quick check to see if `content` is non-empty before writing out the file.
I was testing this locally and think the first 5-6 locales I clicked were empty .HTML files. Only writing the file if there is content would make the `grunt htmllint:{whatever}` target much faster since it would only have to try and read about 1/2 the files.